src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+}
+
+body {
+  font: 11px Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 30px 14px 15px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__photo {
+  height: 134px;
+  width: 160px;
+  align-self: center;
+}
+
+.card__description {
+  font-size: 12px;
+  font-weight: 500;
+  margin-top: 45px;
+  line-height: 15px;
+}
+
+.card__code {
+  margin-top: 5px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.rang {
+  display: flex;
+}
+
+.rang__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background: url(images/star-active.svg) no-repeat center;
+}
+
+.rang__star:last-child {
+  background-image: url(images/star.svg);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 20px;
+}
+
+.card__amount {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__link {
+  margin-top: 20px;
+  background: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  text-align: center;
+  line-height: 40px;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.card__link:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
Set transparent border in .card__link and change only its color on hover

Reply:
Thanks! The value "transparent" is new to me)